migration/src/spreadsheets.rs

```diff
@@ -145,10 +146,11 @@ impl SheetResult {
         let second_value = row
             .next()
             .ok_or_else(|| anyhow::format_err!("Missing second value"))?;
-        let mut is_intro = true;
+        // 0 for Intro, 1 for Body, 2 for Credit
+        let mut chapter_type = 0;
         for cur_row in row {
             if cur_row[0].is_empty() {
-                is_intro = false;
+                chapter_type = chapter_type + 1;
```

Comment:
If I'm understanding this code right, when `cur_row[0]` is empty, then that defines the start of a new chapter type? If I'm not understanding that correctly, is it possible for `chapter_type` to ever exceed 2 and cause issues?

Reply:
cur_row[0] represents the Chapter Depth column in the spreadsheet (https://docs.google.com/spreadsheets/d/12R07Ks8A5g2jffqoILJw7nRn6GehwNwdwosxaJBwNNM/edit?usp=sharing). Each chapter type is separated by the yellow highlighted row, and Chapter Depth is required for every chapter so yes, it should only ever be empty when a new type is being defined. As the spreadsheet is formatted right now I do not believe it will ever be possible for chapter_type to exceed 2, but I thought this strategy might make it easier to add new chapter types and handle that during ingestion if we ever need to

Reply:
^^ let me know if this makes sense or if you still have questions!! thanks for your comments! just committed again with the fixes you suggested

Reply:
yes that makes sense! Sounds good to me!